templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,293 @@
 
-**Import options**
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_slack_data /tmp/slack_data.zip --token <token> --output /tmp/converted_slack_data
+    ./manage.py import <subdomain> /tmp/converted_slack_data
+    ./scripts/start-server
+    ```
 
-The commands above create an imported organization on the root domain
-(`EXTERNAL_HOST`) of the Zulip installation. You can also import into a
-custom subdomain, e.g. if you already have an existing organization on the
-root domain. Replace the last line above with the following, after replacing
-`<subdomain>` with the desired subdomain.
+    !!! tip ""
+        The server stop/restart commands are only necessary when
+        importing on a server with minimal RAM, where an OOM kill might
+        otherwise occur.
 
-```
-./manage.py import <subdomain> converted_slack_data
-```
+1. Follow [step 4](https://zulip.readthedocs.io/en/stable/production/install.html#step-4-configure-and-use)
+   of the guide for [installing a new Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html).
 
-### Remove the Slack app used for export
+{end_tabs}
+
+#### Import details
+
+Whether you are using Zulip Cloud or self-hosting Zulip, here are few notes to keep
+in mind about the import process:
+
+- Slack does not export organization settings, so you will need to [set up your
+  Zulip organization](/help/getting-your-organization-started-with-zulip).
 
-Once the import is complete, you should delete [the Slack
-app](https://api.slack.com/apps) (and thus API token) that you created
-in the earlier step.  This will prevent the token from being used to
-access your Slack instance in the future.
+- Slack does not export user settings, so users in your organization may also
+  want to [customize their account settings](/help/getting-started-with-zulip).
 
-{!import-login.md!}
+- Slack's user roles are mapped to Zulip's [user
+  roles](/help/roles-and-permissions) in the following way:
 
-## Caveats
+| Slack role              | Zulip role    |
+|-------------------------|---------------|
+| Workspace Primary Owner | Owner         |
+| Workspace Owner         | Owner         |
+| Workspace Admin         | Administrator |
+| Member                  | Member        |
+| Single Channel Guest    | Guest         |
+| Multi Channel Guest     | Guest         |
+| Channel creator         | none          |
 
-- Slack doesn't export private channels or direct messages unless you pay
-  for Slack Plus or contact Slack support. See
-  [Slack's documentation](https://get.slack.help/hc/en-us/articles/204897248-Guide-to-Slack-import-and-export-tools)
-  for more details.
+- Messages in threads are imported, but they are not explicitly marked as
+  being in a thread.
 
-- (Slack Plus import) Message edit history is currently not imported.
+- Message edit history and `@user joined #channel_name` messages are not imported.
```

Comment:
Perhaps we should also mention that Slack Threads are not supported? Someone asked about that recently. Ref https://github.com/zulip/zulip/issues/9006

Reply:
The current version has this point just above:

> - Messages in threads are imported, but they are not explicitly marked as
  being in a thread.